pilot/pkg/proxy/envoy/v2/mesh_discovery.go

```diff
@@ -15,10 +15,6 @@
 package v2
```

Comment:
You can rename this file - something like grpc_server.go

Reply:
Let's keep it mesh_discovery. Calling it gRPC server is misleading because gRPC server is way too generic. If we were to use that rationale then it's actually a Http2Server etc... What this file actually does is tie the internal data structures required for discovery to the gRPC server.

Also we have way too many battles to fight. Unless I see tangible progress on multi-cluster I am disinclined to be a hamster on wheels doing endless variable and file renaming. I'd like to address other serious matters like why it's taking us > 10 days to figure out why E2E tests are not working. I'd bet that the time reviewers are spending finding these nit picks can well be used to address the broken build and test infrastructure.